app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt

```diff
@@ -287,29 +277,23 @@ class TopicLessonsFragmentTest {
   }
 
   @Test
-  // TODO(@973): Fix TopicLessonsFragmentTest
-  @Ignore
   fun testLessonsPlayFragment_loadRatiosTopic_chapterListIsNotVisible() {
     launch<TopicActivity>(createTopicActivityIntent(internalProfileId, RATIOS_TOPIC_ID)).use {
-      onView(
-        atPositionOnView(R.id.story_summary_recycler_view, 1, R.id.chapter_recycler_view)
-      ).check(
-        matches(not(isDisplayed()))
-      )
+      onView(withId(R.id.chapter_recycler_view)).check(doesNotExist())
```

Comment:
This is a nice improvement: it seems like the new view pager is more memory efficient than the old one if the views aren't even in the hierarchy.